.editorconfig

```diff
@@ -203,4 +203,6 @@ ij_yaml_keep_indents_on_empty_lines = false
 ij_yaml_keep_line_breaks = true
 ij_yaml_space_before_colon = false
 ij_yaml_spaces_within_braces = true
-ij_yaml_spaces_within_brackets = true
\ No newline at end of file
+ij_yaml_spaces_within_brackets = true
+tab_width = 2
```

Comment:
Is this change on purpose? It will change the tab size of all our code, not only YAML.

Reply:
Hm... that's in the yaml/yml section though - does intellij not respect the extensions in .editorconfig?

Reply:
Ah wait, you are totally right. Didn't see that part of the diff, was collapsed. All good!